pom.xml

```diff
@@ -213,6 +213,17 @@
                 <groupId>org.antipathy</groupId>
                 <artifactId>mvn-scalafmt_${scala.major.version}</artifactId>
                 <version>${mvn-scalafmt.version}</version>
+                <configuration>
+                    <validateOnly>true</validateOnly>
+                    <configLocation>.scalafmt.conf</configLocation>
+                </configuration>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>format</goal>
```

Comment:
Two issues here:
1. can't simply run `mvn scalafmt:format` in order to format code locally;
2. can't compile files without formatting them.

I suggest to remove `validateOnly` and running on format goal. Instead, we should check formatting on CI using github action, similar to https://github.com/evis/scalafix-maven-plugin/blob/master/.github/workflows/scalafix.yml.

This way, the issues will be solved, and formatting will still be checked on CI.